cmd/login_cloud.go

```diff
@@ -14,37 +16,86 @@ var loginCloudCommand = &cobra.Command{
 	Short: "Authenticate with Load Impact",
 	Long: `Authenticate with Load Impact.
 
-This will set the default server used when just "-o cloud" is passed.`,
+This will set the default token used when just "k6 run -o cloud" is passed.`,
+	Example: `
+  # Show the stored token.
+  k6 login cloud -s
+
+  # Store a token.
+  k6 login cloud -t YOUR_TOKEN
+
+  # Log in with an email/password.
+  k6 login cloud`[1:],
 	Args: cobra.NoArgs,
 	RunE: func(cmd *cobra.Command, args []string) error {
 		config, cdir, err := readDiskConfig()
 		if err != nil {
 			return err
 		}
 
-		conf := config.Collectors.Cloud
-		form := ui.Form{
-			Fields: []ui.Field{
-				ui.StringField{
-					Key:     "token",
-					Label:   "API Token",
-					Default: conf.Token,
-				},
-			},
-		}
-		vals, err := form.Run(os.Stdin, stdout)
+		flags := cmd.Flags()
+		show := getNullBool(flags, "show")
+		token, err := flags.GetString("token")
 		if err != nil {
 			return err
 		}
-		if err := mapstructure.Decode(vals, &conf); err != nil {
-			return err
+
+		printToken := func(conf cloud.Config) {
+			fmt.Fprintf(stdout, "  token: %s\n", ui.ValueColor.Sprint(conf.Token))
+		}
+		conf := config.Collectors.Cloud
+
+		switch {
+		case show.Bool:
+		case token != "":
+			conf.Token = token
+		default:
+			printToken(conf)
```

Comment:
Drop this, and the `printToken()` function, just always do the `fmt.Fprintf` call normally at the end.